bin/marked

```diff
@@ -23,9 +23,9 @@ function help() {
     customFds: [0, 1, 2]
   };
 
-  spawn('man', [__dirname + '/../man/marked.1'], options)
-    .on('error', function(err) {
-      fs.readFile(__dirname + '/../man/marked.1.txt', 'utf8', function(err, data) {
+  spawn('man', [path.resolve(__dirname, '/../man/marked.1')], options)
+    .on('error', function(err) { // eslint-disable-line handle-callback-err
```

Comment:
Should this `err` be removed since it is unused rather than disabled eslint here?

Reply:
good call